### PR TITLE
Small change in disc_extract

### PR DESCRIPTION
**Pull request by matt4te**
Removed an option to the ffmpeg call which was causing errors

### CODE REVIEW

File: R/disc_extract.R [hunk 1]
```diff
@@ -189,7 +189,7 @@ disc_extract.goproVideo <- function(data, start, stop, dir, verbose=FALSE, fps=1
       }
       
       # cut the source video
-      exit <- system2("ffmpeg", str_c(" -accurate_seek ", start_offset, " -i \"", x$file, "\" ", stop_offset, " -c copy -an \"", x$temp_video_file, "\""), stdout=FALSE, stderr=ifelse(verbose, "", FALSE))
```
Comment (jiho) [inline R/disc_extract.R]: What was the problem with this? `-an` is supposed to remove the audio, which makes files smaller, encoding faster and will allow to later mux the audio from the sound recorder if needed.

**Comment by matt4te** [comment]
I am sorry. This was terribly sloppy of me. I meant to remove this -an
option from the AUDIO splitting. It is clear why we don't want this option
for splitting a file which is only audio.

I can revert these changes and submit a new pull request, if you like.


-
Matthew Foretich
PhD Candidate | The University of Miami
Rosenstiel School of Marine and Atmospheric Science
mforetich@rsmas.miami.edu
678-467-2256 <(678)%20467-2256>

**Comment by jiho** [comment]
Just make the appropriate changes, commit them with `--amend` and `git push -force` and it will replace the code in this pull request.

**Comment by matt4te** [comment]
I had already reverted the commit in my master, so I think I just need to
make a new commit with the correct changes and then push. Let me know if I
don't do this correctly.


-
Matthew Foretich
PhD Candidate | The University of Miami
Rosenstiel School of Marine and Atmospheric Science
mforetich@rsmas.miami.edu
678-467-2256

**Pull request merged**